composer.json

```diff
@@ -146,11 +146,7 @@
     "autoload": {
         "psr-0": {
             "Sulu\\": "src/"
-        },
-        "exclude-from-classmap": [
-            "src/Sulu/Component/*/Tests/",
-            "src/Sulu/Bundle/*/Tests/"
```

Comment:
Are you sure we should remove that? Doesn't that slow down everything in production?